launch/pipeline/service.py

```diff
@@ -1,18 +1,38 @@
 import inspect
+import uuid
 from typing import Any, Callable, Dict, List, Optional
 
 from launch.pipeline.deployment import Deployment
 from launch.pipeline.runtime import Runtime
 
+MAX_SERVICE_NAME_SIZE = 50
+
 
 class ServiceDescription:
```

Comment:
Also I think this is one of the classes that gets instantiated on the inference endpoint itself, think it would help to document this if that is the case